cpp/oneapi/dal/backend/primitives/ndarray.hpp

```diff
@@ -177,6 +177,7 @@ class ndview : public ndarray_base<axis_count, order> {
 
 public:
     using base = ndarray_base<axis_count, order>;
+    using this_t = ndview<T, axis_count, order>;
```

Comment:
I don't see any benefit of introducing such type, why you can't use ndview like in all other methods (i.e. wrap) instead?